src/react/src/components/ClaimFacilityIntroStep.jsx

```diff
@@ -1,79 +1,171 @@
+                    Owners and senior managers can claim their production
+                    locations on OS Hub by submitting information that allows
+                    the OS Hub team to confirm the production location’s name
+                    and address AND that the person claiming the production
+                    location is affiliated with it. Once a claim is approved,
+                    additional details can be displayed on the production
+                    location’s OS Hub profile. All data contributed by a
+                    claimant will have the “claimed” icon{' '}
+                    <BadgeClaimed
+                        fontSize="18px"
+                        viewBox="0 -5 16 20"
+                        overflow="visible"
+                    />{' '}
+                    displayed next to it on the OS Hub profile.
+                </Typography>
+            </div>
+            <div style={paragraphStyle}>
+                <Typography variant="heading" style={boldStyle}>
+                    To complete a claim request, you will need to submit the
+                    following documentation:
+                </Typography>
+                <Typography variant="subheading">
+                    <ul style={listStyle}>
+                        <li>
+                            A document or website that lists the name and
+                            address of the production location (e.g. utility
+                            bill, business website, registration document, or
+                            LinkedIn profile)
+                        </li>
+                        <li>
+                            A document or website that shows your role at the
+                            production location (e.g. business website,
+                            employment badge, letter of employment, or other
+                            relevant employment documentation){' '}
+                        </li>
+                    </ul>
+                </Typography>
+            </div>
+            <div style={paragraphStyle}>
+                <Typography
+                    variant="heading"
+                    style={{ ...boldStyle, ...inlineStyle }}
+                >
+                    Note:
+                </Typography>
+                &nbsp;
+                <Typography variant="subheading" style={inlineStyle}>
+                    Any documentation appearing to be forged or counterfeit may
+                    result in your claim request being denied.
+                </Typography>
+            </div>
+            <FormControl>
+                <RadioGroup
+                    aria-labelledby="demo-controlled-radio-buttons-group"
+                    name="controlled-radio-buttons-group"
+                    value={selectedValue}
+                    onChange={handleChange}
+                >
+                    <FormControlLabel
+                        value="true"
```

Comment:
Use value="yes" instead of truthy value.

Reply:
done